eogrow/pipelines/rasterize.py

```diff
@@ -252,5 +255,9 @@ def _get_dataset_path(self, filename: str) -> str:
     def _get_output_features(self) -> List[FeatureSpec]:
         """Lists all features that are to be saved upon the pipeline completion"""
         features: List[FeatureSpec] = [FeatureType.BBOX]
+
+        if self.vector_feature[0].is_temporal():
```

Comment:
As you mentioned, you're bothered by the `self.vector_feature[0].is_temporal()`, which I agree with. The only thing I can come up with is defining the following

```python
f_type, _ = self.vector_feature
```

and then using `f_type.is_temporal()`. No need to take this into account though. Approving, but I'm letting you decide.

Reply:
another option is to have a static method `vector_feature_is_temporal(feature)` and hide that in there.
Sure it's a one-liner, but if it makes the code easier to read...

Reply:
can do oneliner static method, but wasn't sure where to add it; should it be in the class, or in utils, or ....

Reply:
I would either add it as part of the class, or add it as a utility function at the end of the file with a `_` prefix so it's not in the docs.